courses/free-courses-hi.md

```diff
@@ -276,3 +278,15 @@
 ### Rust
 
 * [Rust Complete Tutorial In Hindi](https://www.youtube.com/playlist?list=PLRuqvIc0eAmp8Lv6M4BKQWEinvCuqdVFP) - One Two Coding
+
+
+### Software Engineering
+
+* [Software Engineering\| Gate Smashers](https://www.youtube.com/playlist?list=PLxCzCOWd7aiEed7SKZBnC6ypFDWYLRvB2) - Gate Smashers (YouTube)
```

Comment:
- Don't invent titles. Use the one provided by resource...
- YT videos/playlist don't need annotations

```suggestion
* [Software Engineering](https://www.youtube.com/playlist?list=PLxCzCOWd7aiEed7SKZBnC6ypFDWYLRvB2) - Gate Smashers
```

Reply:
titles corrected and YT annotations removed